src/storage/BrowserStorage.ts

```diff
@@ -0,0 +1,162 @@
+import { IStorage } from './IStorage';
```

Comment:
`BrowserStorage.ts` shares lots of code with `NodeFileStorage.ts`. I don't like that. We could use some kind of base class that provides the functionality for storing events in a key-value-store, which the filesystem and local storage both are.